code/modules/reagents/chemistry/reagents/medicine.dm

```diff
@@ -856,7 +856,7 @@
 	var/update_flags = STATUS_UPDATE_NONE
 	M.SetSlur(0)
 	M.AdjustDrunk(-4)
```

Comment:
_Yes 👀_
Could you also fix the side-effects-purging part of Antihol? I believe it's supposed to be `M.AdjustDrunk(-8 SECONDS)` rather than `M.AdjustDrunk(-4)`

Reply:
Sure. I'll go ahead and do this too.